### ISSUE TITLE
aria-label inside table heading

### ISSUE THREAD

**Issue by adamsilver** [adamsilver/f#187]
IE11/JAWS/windows 8.1—doesn’t work, announces “su”
IE9/JAWS/windows 7—doesn’t work, announces “s” “u”
FF/NVDA/Windows 7—works, announces “Sunday”
Safari/VoiceOver—doesn’t announce header at all (check)

Support is patchy, but the date picker does work in FF/NVDA as an example.
